### PR TITLE
Update links for Kubeflow Training Operator

### PR DESCRIPTION
**Pull request by andreyvelich**
After this PR: https://github.com/kubeflow/training-operator/issues/1348, we renamed the repo to `training-operator`.
I tried to update all corresponding links and update some legacy docs for Training.

Please take a look.

I didn't update [References](https://www.kubeflow.org/docs/reference/) docs, do we want to keep them ?
We have some scripts to generate it: https://github.com/kubeflow/website/tree/master/gen-api-reference.

/assign @kubeflow/wg-training-leads @shannonbradshaw

### CODE REVIEW

**Comment by shannonbradshaw** [comment]
Thanks for submitting this PR, @andreyvelich! I will review first thing in my morning tomorrow.

**Review by terrytangyuan — APPROVED** [review]
/lgtm

File: content/en/docs/components/training/mxnet.md [hunk 1]
```diff
@@ -101,18 +101,18 @@ should be created for each replica.
...
+kubectl create -f https://raw.githubusercontent.com/kubeflow/training-operator/master/examples/mxnet/tune/mx_job_tune_gpu_v1.yaml
 ```
 
 Before you use the auto-tuning example, there is some preparatory work need to be finished in advance.
 To let TVM tune your network, you should create a docker image which has TVM module.
 Then, you need a auto-tuning script to specify which network will be tuned and set the auto-tuning parameters.
 For more details, please see [tutorials](https://docs.tvm.ai/tutorials/autotvm/tune_relay_mobile_gpu.html#sphx-glr-tutorials-autotvm-tune-relay-mobile-gpu-py).
-Finally, you need a startup script to start the auto-tuning program. In fact, mxnet-operator will set all the parameters as environment variables and the startup script need to reed these variable and then transmit them to auto-tuning script.
+Finally, you need a startup script to start the auto-tuning program. In fact, MXJob will set all the parameters as environment variables and the startup script need to reed these variable and then transmit them to auto-tuning script.
```
Comment (shannonbradshaw) [inline content/en/docs/components/training/mxnet.md:115]: Fixes in bold below.
Finally, you need a startup script to start the auto-tuning program. In fact, MXJob will set all the parameters as environment variables and the startup script **needs** to **read** these variable and then transmit them to **the** auto-tuning script.

File: content/en/docs/components/training/xgboost.md [hunk 2]
```diff
@@ -44,34 +50,42 @@ NAME                                READY   STATUS    RESTARTS   AGE
 training-operator-d466b46bc-xbqvs   1/1     Running   0          4m37s
 ```
 
-## Creating a XGBoost Job
+## Creating a XGBoost training job
 
-You can create XGBoost Job by defining a XGboostJob config file. See the manifests for the [IRIS example](https://github.com/kubeflow/tf-operator/blob/master/examples/xgboost/xgboostjob.yaml). You may change the config file based on your requirements. eg: add `CleanPodPolicy` in Spec to `None` to retain pods after job termination.
+You can create a training job by defining a `XGboostJob` config file. See the
+manifests for the [IRIS example](https://github.com/kubeflow/training-operator/blob/master/examples/xgboost/xgboostjob.yaml).
+You may change the config file based on your requirements. eg: add `CleanPodPolicy`
```
Comment (shannonbradshaw) [inline content/en/docs/components/training/xgboost.md:57]: eg: -> E.g.,

**Review by shannonbradshaw — APPROVED** [review]
I suggested corrections in few files, otherwise. /lgtm

**Comment by thesuperzapper** [comment]
FYI, the PR https://github.com/kubeflow/website/pull/3014 restructures the `Components / Training Operators` section to allow for moving `References` from the top level.

I don't mind who merges first, but someone will have to rebase.

**Comment by terrytangyuan** [comment]
> I think we can remove reference for Training Operators in the future, since we keep them here: https://github.com/kubeflow/training-operator/tree/master/docs/api.
> 
> WDYT @kubeflow/wg-training-leads ?
 
Yes. It's easy to get outdated on the website. Also there's an issue to autogenerate the docs https://github.com/kubeflow/website/issues/1924.

**Comment by andreyvelich** [comment]
> > I think we can remove reference for Training Operators in the future, since we keep them here: https://github.com/kubeflow/training-operator/tree/master/docs/api.
> > WDYT @kubeflow/wg-training-leads ?
> 
> Yes. It's easy to get outdated on the website. Also there's an issue to autogenerate the docs #1924.

Sounds good.
@kubeflow/wg-training-leads If you are fine with these changes, I think we can merge this PR.

**Comment by andreyvelich** [comment]
@Bobgy @james-jwu @zijianjoy Please can you help with this PR approval ?

**Pull request merged**